doc/herbstluftwm.txt

```diff
@@ -398,6 +399,7 @@ split 'ALIGN' ['FRACTION']::
         * split horiz 0.3
         * split vertical 0.5
         * split h
+        * split top 0.2 ''   (splits the root frame)
```

Comment:
So an empty frame index means root frame? Is this "known"? ;)

Reply:
Yes, it's documented in the section "frame index" (I've added a reference)

Reply:
Awesome, thank you!